### PR TITLE
News on SARS-CoV2 variant Competition Dashboard

### PR DESCRIPTION
**Pull request by HamsaiyniVenkatesh**
Updated the content on md file, includes images for thumbnail and content graph.

### CODE REVIEW

File: data/available_datasets.json [hunk 1]
```diff
@@ -1,6 +1,543 @@
 {
   "last_updated": "2024-06-10",
```
Comment (LianeHughes) [inline data/available_datasets.json:2]:
```suggestion
  "last_updated": "2024-09-04",
```

**Review by LianeHughes — CHANGES_REQUESTED** [review]
On the available data file, please ensure that the data is from the publication itself. Correction is needed for the spelling of 'protein' (in data type and database name), and the tag for protein data should align to what is used in the rest of the file (I think 'Protein data', please check). Description text should also use British English spelling.

**Comment by LianeHughes** [comment]
and publications file should be uploaded

**Comment by HamsaiyniVenkatesh** [comment]
The publication file is added in a right branch with details for the task number ( as added the data here in this branch was a mistake) , the publication file is in the Hamsa/Freya- 833,834

**Review by LianeHughes — APPROVED** [review]
corrections added

**Pull request merged**